src/import/sst_service.rs

```diff
@@ -574,13 +591,17 @@ impl<E: Engine> ImportSst for ImportSstService<E> {
                 CONFIG_ROCKSDB_GAUGE.with_label_values(&[cf, name]).set(v);
             }
 
-            match req.get_mode() {
-                SwitchMode::Normal => self
-                    .importer
-                    .enter_normal_mode(self.tablet_registry.clone(), mf),
-                SwitchMode::Import => self
-                    .importer
-                    .enter_import_mode(self.tablet_registry.clone(), mf),
+            if let LocalTablets::Singleton(tablet) = &self.tablets {
+                match req.get_mode() {
+                    SwitchMode::Normal => self.importer.enter_normal_mode(tablet.clone(), mf),
+                    SwitchMode::Import => self.importer.enter_import_mode(tablet.clone(), mf),
+                }
+            } else if req.get_mode() != SwitchMode::Normal {
+                Err(sst_importer::Error::Engine(
+                    "partitioned-raft-kv doesn't support import mode".into(),
```

Comment:
Is it the temporary behavior?
It's a performance blow if partitioned-raft-kv cannot support it

Reply:
It's not supported due to complexity. Whether the performance is good enough depends on test.

Reply:
> It's not supported due to complexity. Whether the performance is good enough depends on test.
What's the exact complexity with import mode? Could you elaborate it.  I think SST ingest probably is faster than batch insert, regardless it's multi-rocksdb or not.

Reply:
No matter whether it's import mode, all data are imported into rocksdb as SSTs. The difference is whether tun rocksdb to skip flow controls like L0 file count. The complexity comes from rocksdb is changed from time to time, setting configuration it is hard to be implemented right.